src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -95,12 +83,26 @@ export default function Filters({ page }: FilterProps) {
               /** A numeric value associated with the event (e.g. 42) */
             })
           }}
-          defaultValue={getFilters(filter_type)}
+          defaultValue={pageState.filters[filter_type] as string[]}
         />
       </div>
     )
   }
 
+  const buildFilterCheckbox = (filter_type: FilterType, label: string) => {

```

Comment:
Not sure how this worked before, but what is this used for & why was this added?

Reply:
This is a new function that renders a checkbox filter rather than our typical dropdown filter. Needed to create this for the unity studies checkbox that was newly implemented